app/src/main/java/com/mertceyhan/bitcoinmarket/components/Chart.kt

```diff
@@ -18,6 +20,7 @@ import com.mertceyhan.bitcoinmarket.utils.extensions.setLineDataSet
 fun Chart(
     modifier: Modifier = Modifier,
     lineDataSet: LineDataSet? = null,
+    isShowShimmer: Boolean
```

Comment:
I think `isPlaceHolderVisible` would be more meaningful cause there is no `shimmer` term in `Modifier`. If you agree, could you change all `isShowShimmer` naming to `isPlaceHolderVisible`